src/writing-your-own-application/generate-project/esp-template.md

```diff
@@ -0,0 +1,210 @@
+[2023-04-17T14:17:08Z INFO ] Connecting...
+[2023-04-17T14:17:09Z INFO ] Using flash stub
+[2023-04-17T14:17:09Z WARN ] Setting baud rate higher than 115,200 can cause issues
+Chip type:         esp32c3 (revision v0.3)
+Crystal frequency: 40MHz
+Flash size:        4MB
+Features:          WiFi, BLE
+MAC address:       60:55:f9:c0:39:7c
+App/part. size:    203,920/4,128,768 bytes, 4.94%
+[00:00:00] [========================================]      13/13      0x0
+[00:00:00] [========================================]       1/1       0x8000
+[00:00:01] [========================================]      64/64      0x10000                                                                                                                    [2023-04-17T14:17:11Z INFO ] Flashing has completed!
+Commands:
+    CTRL+R    Reset chip
+    CTRL+C    Exit
+
+ESP-ROM:esp32c3-api1-20210207
+Build:Feb  7 2021
+rst:0x15 (USB_UART_CHIP_RESET),boot:0xc (SPI_FAST_FLASH_BOOT)
+Saved PC:0x40380816
+0x40380816 -
+    at ??:??
+SPIWP:0xee
+mode:DIO, clock div:2
+load:0x3fcd5820,len:0x16ec
+0x3fcd5820 - _stack_start
+    at ??:??
+load:0x403cc710,len:0x95c
+0x403cc710 -
+    at ??:??
+load:0x403ce710,len:0x2dc0
+0x403ce710 -
+    at ??:??
+SHA-256 comparison failed:
+Calculated: 692c10f3e7d531666ff34d02d3da161b3daa41ea629010d031fe3706fbada122
+Expected: 9fafed52ab0387e903bde368d0d6bfffe0dcc3d2f90dca069a4db891108c387c
+Attempting to boot anyway...
+entry 0x403cc710
+0x403cc710 -
+    at ??:??
+I (43) boot: ESP-IDF v5.0-beta1-764-gdbcf640261 2nd stage bootloader
+I (43) boot: compile time 11:30:26
+I (43) boot: chip revision: V003
+I (46) boot.esp32c3: SPI Speed      : 40MHz
+I (51) boot.esp32c3: SPI Mode       : DIO
+I (56) boot.esp32c3: SPI Flash Size : 4MB
+I (61) boot: Enabling RNG early entropy source...
+I (66) boot: Partition Table:
+I (70) boot: ## Label            Usage          Type ST Offset   Length
+I (77) boot:  0 nvs              WiFi data        01 02 00009000 00006000
+I (84) boot:  1 phy_init         RF data          01 01 0000f000 00001000
+I (92) boot:  2 factory          factory app      00 00 00010000 003f0000
+I (99) boot: End of partition table
+I (103) esp_image: segment 0: paddr=00010020 vaddr=3c030020 size=05f74h ( 24436) map
+I (117) esp_image: segment 1: paddr=00015f9c vaddr=40380000 size=007f8h (  2040) load
+I (121) esp_image: segment 2: paddr=0001679c vaddr=00000000 size=0987ch ( 39036)
+I (137) esp_image: segment 3: paddr=00020020 vaddr=42000020 size=21c40h (138304) map
+I (168) boot: Loaded app from partition at offset 0x10000
+I (168) boot: Disabling RNG early entropy source...
+Hello world!
+```
+
+What you see here are messages from the first and second stage bootloader and then ... our "Hello World" message!
+
+And that is exactly what the code is doing.
+
+You can reboot with `CTRL+R` or exit with `CTRL+C`.
+
+In the next chapters, we will be modifying this template to generate some more interesting applications.
```

Comment:
Now that this section has been moved, it is not immediately obvious what the next chapters are. Suggest giving a more explicit hint or a link.